src/ol_openedx_chat/static/css/ai_chat.css

```diff
@@ -13,31 +13,36 @@
 }
 
 .ai-chat-button {
+    all: unset;
     display: flex;
     flex-direction: row;
     position: relative;
     bottom: 70px;
-    left: 70%;
+    left: 72%;
```

Comment:
If messing with these percentages gets annoying, could also consider making the container `display: flex; justify-content: flex-end`.

Reply:
I am skipping this change because:

It works for the normal learners as they do not have extra buttons on the right side.
<img width="1757" alt="Screenshot 2025-03-06 at 1 34 43 PM" src="https://github.com/user-attachments/assets/18284646-5d27-4d52-9354-d04a34a11e83" />

Has issues for admins/instructors as there some more UI elements on the right.
<img width="1757" alt="Screenshot 2025-03-06 at 1 30 12 PM" src="https://github.com/user-attachments/assets/084c9765-16b9-4aed-8066-3f63968a940f" />